acp/identity/identity.go

```diff
@@ -14,22 +14,26 @@ Package identity provides defradb identity.
 
 package identity
 
-import (
-	"github.com/sourcenetwork/immutable"
-)
+// Identity is the unique identifier for an actor.
+type Identity string
 
 var (
-	// NoIdentity is an empty identity.
-	NoIdentity = immutable.None[string]()
+	// None is an empty identity.
+	None = Identity("")
 )
 
-// NewIdentity makes a new identity if the input is not empty otherwise, returns an empty Option.
-func NewIdentity(identity string) immutable.Option[string] {
+// New makes a new identity if the input is not empty otherwise, returns None.
+func New(identity string) Identity {
```

Comment:
question: why not:
```suggestion
func New(identity string) immutable.Option[Identity] {
```

Reply:
I've made identity an `immutable.Option` everywhere again.

Reply:
Thanks buddy